src/include/sof/lib/notifier.h

```diff
@@ -53,8 +65,25 @@ struct notify **arch_notify_get(void);
 void notifier_register(struct notifier *notifier);
 void notifier_unregister(struct notifier *notifier);
 
+void notifier_register_sync(void *receiver, void *caller,
+			    enum notify_sync_id type,
+		void (*cb)(void *arg, enum notify_sync_id type, void *data));
```

Comment:
I guess this weird formatting is caused by the will to satisfy checkpatch...
I think in cases like that it's better to just indent all lines with 1 tab instead of having pyramids like this ;)

Reply:
done